doc/source/tutorials/level-5-tutorial.rst

```diff
@@ -296,6 +296,10 @@ Please refer to :ref:`faq-doc-pr-preview`
 
 .. include:: ../snippets/api-reference-doc.rst
 
+(Optional) Set up default prompt values
```

Comment:
This is the first tutorial that the user sees. Please be explicit why this is interesting for the user. There should be a quick sales pitch addressing the problem